test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -1139,7 +1139,7 @@ public void ProcessResults_ValueTypePropertyWithBindRequired_RequiredValidatorIg
                 .Value;
             var error = Assert.Single(entry.Errors);
             Assert.Null(error.Exception);
-            Assert.Equal("A value for the 'ValueTypeProperty' property was not provided.", error.ErrorMessage);
+            Assert.Equal("A value for the 'theModel.ValueTypeProperty' property was not provided.", error.ErrorMessage);
```

Comment:
Doesn't this make these error messages deviate from what things like `LabelFor(...)` would generate? Let's chat tomorrow, I'm not convinced that adding the prefix into these messages helps the end user.